src/Core/Bridge/Symfony/Bundle/Action/SwaggerUiAction.php

```diff
@@ -56,6 +56,7 @@ final class SwaggerUiAction
     private $oauthTokenUrl;
     private $oauthAuthorizationUrl;
     private $oauthScopes;
+    private $oauthPKCE;
```

Comment:
```suggestion
    private $oauthPkce;
```

(Same to do everywhere to respect our CS).